frontend/templates/apply/apply-base.html.jinja

```diff
@@ -0,0 +1,86 @@
+{% extends "fs-base.html.jinja" %}
+
+<meta name="description" content="Apply for a Grant">
+<meta name="keywords" content="Apply for a Grant">
+<meta name="author" content="Ministry of Housing, Communities and Local Government">
```

Comment:
I don't think these will work - when extending templates, I think everything has to be inside blocks or they won't come out in the final HTML?

Reply:
Moved them into a block